crypto/dilithium/pqcrystals_dilithium_ref_common/polyvec.c

```diff
@@ -40,33 +42,56 @@ void polyvec_matrix_pointwise_montgomery(ml_dsa_params *params,
                                          const polyvecl *mat,
                                          const polyvecl *v) {
   unsigned int i;
-
-  for(i = 0; i < params->k; ++i)
+  for(i = 0; i < params->k; ++i) {
     polyvecl_pointwise_acc_montgomery(params, &t->vec[i], &mat[i], v);
+  }
 }
 
 /**************************************************************/
 /************ Vectors of polynomials of length L **************/
 /**************************************************************/
 
+/*************************************************
+* Name:        polyvecl_uniform_eta
+*
+* Description: FIPS 204: Algorithm 33 ExpandS (for vectors l).
+*              Samples vector v with polynomial coordinates whose
+*              coefficients are in [-eta, eta].
+*
+* Arguments:   - ml_dsa_params: parameter struct
+*              - polyvecl v: pointer to input vector
```

Comment:
```suggestion
*              - polyvecl v: pointer to output vector
```

Reply:
fixed in [a8777cd](https://github.com/aws/aws-lc/pull/2017/commits/a8777cd91adbc6ddacbc0de027fce11eb8d3b4e7)